AllJoyn/Platform/Bridge/BridgeCore/Bridge.vcxproj

```diff
@@ -288,8 +289,10 @@
       <Optimization>MaxSpeed</Optimization>
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <IntrinsicFunctions>true</IntrinsicFunctions>
-      <PreprocessorDefinitions>QCC_OS_GROUP_WINDOWS;NDEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>BRIDGE_WIN10;QCC_OS_GROUP_WINDOWS;_SCL_SECURE_NO_WARNINGS;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Debug already builds fine. Release was not building. Same is true for al the projects.

---
In reply to: [75174093](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75174093) [](ancestors = 75174093)